dces-drc-integration/src/test/java/uk/gov/justice/laa/crime/dces/integration/service/ContributionServiceTest.java

```diff
@@ -84,13 +84,14 @@ void testXMLValid() throws JAXBException {
 		when(contributionsMapperUtils.generateFileXML(any(), any())).thenReturn("ValidXML");
 		when(contributionsMapperUtils.generateFileName(any())).thenReturn("TestFilename.xml");
 		when(contributionsMapperUtils.generateAckXML(any(), any(), any(), any())).thenReturn("ValidAckXML");
+		when(feature.noOfContributionRecords()).thenReturn(5);
```

Comment:
Maybe there coul be two tests for the paging?

One for when the response fits in one request (i.e. the old response with just a couple of records) being less than 5.

And then a new test for when there is more than one "page" of records (which is this new code with 8 records). What do you think?